device-types/TrippLite/SMART3000RMXL2U.yaml

```diff
@@ -0,0 +1,72 @@
+manufacturer: Tripplite
+model: SMART3000RMXL2U
+slug: smart3000rmxl2u
+part_number: ''
+u_height: 2.0
+is_full_depth: true
+subdevice_role: ''
+airflow: front-to-rear
+comments: ''
+power-ports:
+- name: Plug1
+  type: nema-l5-30p
+  maximum_draw: 2880
+  allocated_draw: null
+  label: ''
+  description: UPS
+power-outlets:
+- name: Outlet1-1
+  type: nema-5-15r
+  power_port: Plug1
+  feed_leg: ''
+  label: ''
+  description: ''
+- name: Outlet1-2
+  type: nema-5-15r
+  power_port: Plug1
+  feed_leg: ''
+  label: ''
+  description: ''
+- name: Outlet1-3
+  type: nema-5-15r
+  power_port: Plug1
+  feed_leg: ''
+  label: ''
+  description: ''
+- name: Outlet1-4
+  type: nema-5-15r
+  power_port: Plug1
+  feed_leg: ''
+  label: ''
+  description: ''
+- name: Outlet2-1
+  type: nema-5-15r
+  power_port: Plug1
+  feed_leg: ''
+  label: ''
+  description: ''
+- name: Outlet2-2
+  type: nema-5-15r
+  power_port: Plug1
+  feed_leg: ''
+  label: ''
+  description: ''
+- name: Outlet2-3
+  type: nema-5-15r
+  power_port: Plug1
+  feed_leg: ''
+  label: ''
+  description: ''
+- name: Outlet2-4
+  type: nema-5-15r
+  power_port: Plug1
+  feed_leg: ''
+  label: ''
+  description: ''
+- name: PDU Outlet1
```

Comment:
This does not seem like an official outlet name

Reply:
I checked the back of the device, the datasheet, and the product page. It's not labeled. Changed the name to Outlet 3